src/shared/components/MultiFileSelect/MultiFileSelect.tsx

```diff
@@ -206,13 +206,12 @@ export const MultiFileSelect: React.FC<MultiFileSelectProps> = ({
         <Step
           variant="file"
           number={1}
-          title={files.video ? 'Video file' : 'Add video file'}
-          active={step === 'video'}
+          title={files.video ? (files.video.blob ? (files.video.blob as File).name : 'Video file') : 'Add video file'}
+          active={step === 'video' && stepsActive}
           stepPlaceholder={!!files.video && <SvgGlyphFileVideo />}
           disabled={editMode}
           completed={!!files.video}
           onDelete={() => handleDeleteFile('video')}
-          onClick={() => handleChangeStep('video')}
           isLoading={isLoading}
         />
         <StepDivider>
```

Comment:
Btw, we should add also step divider on mobile view: 
![image](https://user-images.githubusercontent.com/51168865/133662043-d74ef23a-cce1-4ea5-bfb6-f822dc38e287.png)

Reply:
@KubaMikolajczyk how divider's animation should work on mobile?